ChangeLog

```diff
@@ -1,3 +1,20 @@
+2021-03-03 Riccardo Mottola <rm@gnu.org>
```

Comment:
When doing a pull request it is better to squash the ChangeLog entries into one.